components/table/table-col-sort-button.js

```diff
@@ -27,13 +27,21 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 			nosort: {
 				reflect: true,
 				type: Boolean
+			},
+			/**
+			 * Check to see if the button has siblings.
+			 * @type {boolean}
+			 */
+			hasSiblings: {
```

Comment:
I'd recommend `hasSibling` instead since it only matters if they have at least one.
Also we'd want the doc comments for this to just be `@ignore` since we wouldn't want internal consumers setting this.
See [Daylight > Attributes & Properties](https://daylight.d2l.dev/developing/best-practices/attributes-properties/#do%3A-use-correct-casing-for-attributes-and-properties) for details on defining this property and corresponding attribute.